housingbayarea/bloom
# fix: allow sms mfa to be toggled on and off

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

Due to a change in Twilio, our SMS messages are failing about 50% of the time. We are working to get this resolved, but it could take a few weeks. In the meantime we plan on turning off the option of using SMS as the 2fa method.

It will look like this 
![image](https://github.com/housingbayarea/bloom/assets/42942267/69f0eebb-f747-4fd0-8846-a9c425dcf2ea)


## How Can This Be Tested/Reviewed?

in the `.env` file in partners add the following which will toggle the sms functionality off.
`SHOW_SMS_MFA=FALSE`

You will also need to have an account that has MFA enabled. The easiest way to do that is reseed and create a new account on the partner site. That new account should have MFA enabled by default

It is also recommended to test it with the above set to `TRUE` or not set at all

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
